source/_docs/automation/templating.markdown

```
@@ -54,26 +60,50 @@ Inherites template variables from [event](#event) or [state](#state) template ba
 
 | Template variable | Data |
 | ---- | ---- |
-| `trigger.platform` | Hardcoded: `device`.
+| `trigger.platform` | Hardcoded: `device`
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)